tests/test_save_load_anthropic.py

```
@@ -979,75 +981,67 @@ def identifiable_tool(test_id: str) -> str:
+            has_first_result or has_second_result,
+            f"Expected to find at least one calculation result in response: {final_response}",
         )
 
-        if not branch_contains_calculation_results:
-            print("CONFIRMED BUG: Branch tools execute but results don't appear in recombined output")
-            # This would indicate the bug - tools execute but results don't get integrated
-        else:
-            print("Branch tool results appear in output - bug may be fixed")
-
-        print(f"Branch tool results contain calculation results: {branch_contains_calculation_results}")
+        print(f"SUCCESS: Tool execution integrated with bot responses")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)